flowzone.yml

```diff
@@ -1305,8 +1323,18 @@ jobs:
               echo "Poetry not used"
               echo "enabled=false" >> $GITHUB_OUTPUT
             fi
+
+            has_package=$(awk -F "=" '/^packages/ {print $2}' pyproject.toml)
```

Comment:
Is it possible for this property to have multiple values? How is that handled?

Reply:
@klutchell  It is - but this just means "all these subfolders/modules are part of the final package to be deployed" which means, poetry will handle bundling all those into a wheel/tgz when we call `poetry publish --build` or `poetry build`. Now the problem I see is that we don't want to deploy every single python/poetry repo we have. My workaround this problem was to use packages (which is usually only used if you want to deploy something and I did not found something like "private" in NPM). I found a single repo of our internal python stuff we don't want to publish and has a "packages", so I need to talk to Genadi regarding that before merging this PR. 

If you have other suggestions of how we could figure it out if the package should/should not be deployed I am open to it too :)

Reply:
Yeah, I don't see any other fields in pyproject.toml that would make sense. I guess the only other option is to add a new boolean to Flowzone. Do we want to reach out for more ideas/opinions first?